swaglyrics/tab.py

```diff
@@ -14,8 +14,12 @@
 def tab():
     # format lyrics for the browser tab template
     global song, artist
-    song = spotify.song()
-    artist = spotify.artist()
+    if app.config.get('song') is not None and app.config.get('artist') is not None:
```

Comment:
`if app.config.get('song') and app.config.get('artist'):` 

this could be shortened to this